pages/homepage/homepage.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <link rel="stylesheet" href="https://component-vishalg8454.netlify.app/components/main.css">
+    <link rel="stylesheet" href="./homepage.css">
+    <script src="https://kit.fontawesome.com/06bd83eb3a.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <div class="grid-wrapper">
+
+        <nav class="navbar">
+            <div class="nav-container">
+                <a class="nav-logo" href="#">E-COMMERCE</a>
+                <div class="nav-search-container">
+                    <input class="nav-search-text" type="text">
+                    <button class="nav-search-button"><i class="fas fa-search"></i>
+                    </button>
+                </div>
+                <button class="btn btn-primary-outline  nav-cta">Login</button>
+                <ul class="nav-list">
+                    <li class="nav-list-items">
+                        <div>
+                            <a> <i class="badge-icon fas fa-heart nav-icon"></i></a>
+                            <div class="badge-text">9</div>
+                        </div>
+                    </li>
+                    <li class="nav-list-items">
+                        <div>
+                            <a><i class="badge-icon fas fa-shopping-cart nav-icon"></i></a>
+                            <div class="badge-text">3</div>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+
+        <div class="carousel">
+            <div class="carousel-container">
+                <img class="carousel-img" src="/assets/FICTION.png">
+                <img class="carousel-img" src="/assets/SELF-HELP.png">
+                <img class="carousel-img" src="/assets/COMIC.png">
+                <img class="carousel-img" src="/assets/SCI-FI.png">
+                <img class="carousel-img" src="/assets/BIOGRAPHY.png">
+                <img class="carousel-img" src="/assets/NOVEL.png">
+            </div>
+        </div>
+
+        <div class="banner">
+            <img class="banner-hero" src="/assets/pexels-pixabay-159711.jpeg">
+            <div class="banner-overlay"></div>
+        </div>
+
```

Comment:
please check the image or its styles again. as on the preview i can see a blue layer in nearly half of the image.

Reply:
That is an intended feature. The blue filter has been overlaid which can be used for displaying text.